backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -662,7 +707,8 @@ public Project deepCopyProjectForUser(String srcProjectID, UserInfo newOwner)
 
     // cloned project
     Project newProject = copyProjectAndUpdateDetails(srcProject, newOwner);
-    insertProject(newProject, newOwner);
+    // TODO: FIX ME
+    // insertProject(newProject, newOwner);
```

Comment:
Please add more detail on what needs to be fixed

Reply:
I will add  description, but this change for the `deepCopyProject` endpoint. so Could you confirm that we used `deepCopyProject` endpoint or not?? if it is not used then we can remove this logic.